### PR TITLE
Re issue #2818 - add codebyte for Python Built-in Reversed Function applied on tuple

### PR DESCRIPTION
**Pull request by DelphCL**
### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@DelphCL This only addresses half the issue yo were assigned. You need to include a codebyte for `https://www.codecademy.com/resources/docs/python/built-in-functions/sorted` in with this PR. Tag me once that's accomplished and I'll do a first review.

**Comment by DelphCL** [comment]
> 



> @DelphCL This only addresses half the issue you were assigned. You need to include a codebyte for `https://www.codecademy.com/resources/docs/python/built-in-functions/sorted` in with this PR. Tag me once that's accomplished and I'll do a first review.

@SSwiniarski , the codebyte for the sorted function was added in a previous PR:"add example 6 to Python Built in Functions - sorted() #2845", which was already approved and merged a few days ago. 

Not sure what I have done wrong here, please let me know if any further correction or action is required, 

Many thanks 

Kind regards

DelphCL

**Comment by SSwiniarski** [comment]
@DelphCL This is why we prefer to have only one PR per issue. It leads to confusion like this. I removed the tag and will review this.

File: content/python/concepts/built-in-functions/terms/reversed/reversed.md [hunk 1]
```diff
@@ -9,12 +9,13 @@ Tags:
...
   - 'Iterators'
   - 'Lists'
+  - 'Tuples'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/analyze-data-with-python'
 ---
 
-Takes in an iterator object, such as a list or string, and returns a reversed iterator object.
+Takes in an iteratable object, such as a list, string, or a tuple and returns a reversed iterator object.
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/reversed/reversed.md:18]:
```suggestion
The **`reversed()`** function takes in an iteratable object, such as a list, string, or a tuple and returns a reversed iterator object.
```

File: content/python/concepts/built-in-functions/terms/reversed/reversed.md [hunk 2]
```diff
@@ -65,3 +66,18 @@ for letter in new:
 
 # Output: desserts
 ```
+
+## Codebyte Example
+
+Tuples are iterable objects too hence `reversed()` can be used on tuples as well.
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/reversed/reversed.md:72]:
```suggestion
Tuples are iterable objects too, hence `reversed()` can be used on tuples as well. 
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DelphCL I did a first review and left some comments.

**Comment by DelphCL** [comment]
> 

@SSwiniarski, many thanks, I have accepted the changes you suggested and committed them. Many thanks.

**Review by SSwiniarski — APPROVED** [review]
@DelphCL Looks good for a second review.

**Comment by DelphCL** [comment]
@SSwiniarski, is there any action left for me to do ? many thanks DelphCL

**Review by caupolicandiaz — APPROVED** [review]
@DelphCL @SSwiniarski Just made a couple of typo edits. Looks good to merge

**Pull request merged**